frontend/src/components/regions/AgileBoard/Presentation.js

```diff
@@ -23,8 +23,12 @@ const useStyles = makeStyles((theme) => ({
     },
   },
 
+  paper: {
+    height: "calc(100vh - 9rem)",
```

Comment:
probably 6rem is more appropriate than 9 rem. Ideally the gap at the top should match the gap at the bottom. We want things to be tidy.